Vic2ToHoI4/Source/Hoi4Outputter/Events/EventsOutputter.cpp

```diff
@@ -4,59 +4,48 @@
 
 
 
-void outputActualEvents(
-	const std::string& eventsFileName,
-	const std::string& eventNamespace,
-	const std::vector<HoI4::Event>& nationalFocusEvents,
-	const Configuration& theConfiguration
-);
-void outputWarJustificationEvents(
-	const std::vector<HoI4::Event>& warJustificationEvents,
-	const Configuration& theConfiguration
-);
-void outputStabilityEvents(
-	const std::map<std::string, HoI4::Event>& stabilityEvents,
-	const std::map<std::string, HoI4::Event>& strikesEvents,
-	const std::map<std::string, HoI4::Event>& mutinyEvents,
-	const Configuration& theConfiguration
-);
+void outputActualEvents(const std::string& eventsFileName,
```

Comment:
I'd still lobby for moving all of these free-floating functions into some container class and moving these specific declarations into relevant .h file.

Reply:
Opened an issue for it. I'll get to it eventually, but more focused on the general cleanup first.